dumbo/src/tcp/connection.rs

```diff
@@ -466,9 +465,12 @@ impl Connection {
         }
 
         if v + current_w > max_w {
-            self.local_rwnd_edge = self.ack_to_send + max_w;
+            let adv = self.ack_to_send + max_w;
```

Comment:
According to the method semantics, `adv` should be `old_rwnd_edge` - `new_rwnd_edge`, which in this case means the block might look something like:
```rust
{
  let old_rwnd_edge = self.local_rwnd_edge; 
  self.local_rwnd_edge = self.ack_to_send + max_w;
  (self.local_rwnd_edge - old_rwnd_edge).0
}
```